crates/cli/src/subcommands/call.rs

```diff
@@ -33,14 +33,7 @@ pub fn cli() -> clap::Command {
                 .conflicts_with("anon_identity")
                 .help("The identity to use for the call"),
         )
-        .arg(
-            Arg::new("anon_identity")
-                .long("anon-identity")
-                .short('a')
-                .conflicts_with("identity")
-                .action(ArgAction::SetTrue)
-                .help("If this flag is present, the call will be executed with no identity provided"),
-        )
+        .arg(common_args::anonymous())
```

Comment:
removed the `.conflicts_with("identity")` because the identity param is already marked as conflicting with this one